next.config.js

```diff
@@ -1,43 +1,79 @@
 const isProd = process.env.APP_ENV === 'prod'
+const isDev = process.env.NODE_ENV === 'development'
 const isExport = process.env.BUILD_OPTION === 'static'
 
 /**
  * @type {import('next').NextConfig}
  */
 const nextConfig = {
   compress: false,
+
   images: {
     loaderFile: './src/templates/common/mediaImage/customLoader.js',
     loader: 'custom',
     unoptimized: true,
   },
+
   reactStrictMode: true,
   swcMinify: true,
   trailingSlash: true,
   assetPrefix: undefined,
   output: isExport ? 'export' : undefined,
+
   // assetPrefix: isProd
   //   ? 'https://s3.us-gov-west-1.amazonaws.com/next-content.www.va.gov/'
   //   : undefined,
-  staticPageGenerationTimeout: 180, //arbitrary; 60 is default but it's too small
+  //arbitrary; 60 is default but it's too small
+  staticPageGenerationTimeout: 180,
+
   experimental: {
-    largePageDataBytes: 512 * 1000, // 512kb, is 128kb by default
+    // 512kb, is 128kb by default
+    largePageDataBytes: 512 * 1000,
+
+    // Enable React 19 optimizations
+    reactCompiler: true,
```

Comment:
Calling this out. I'm not sure how much of the performance boost I saw was due to the compiler, but we can try it without if'n y'like.